coops-ui/src/components/AnnualReport/RegisteredOfficeAddress.vue

```diff
@@ -99,12 +98,22 @@ export default class RegisteredOfficeAddress extends Vue {
   /**
    * The identifier for the legal entity that is to have its addresses retrieved from the API.
    */
-  @Prop({ default: '' }) readonly legalEntityNumber: string
+  @Prop({ default: '' })
+  readonly legalEntityNumber: string
```

Comment:
Much better/correct/consistent formatting - that's what happens when I take code from the web and don't think too much about it!